github/repos_pages.go

```diff
@@ -43,6 +46,14 @@ type PagesBuild struct {
 	UpdatedAt *Timestamp  `json:"updated_at,omitempty"`
 }
 
+// PagesBuild represents the HTTPS Certificate information for a GitHub Pages site.
+type PagesHttpsCertificate struct {
+	State       *string    `json:"state,omitempty"`
+	Description *string    `json:"description,omitempty"`
+	Domains     *[]*string `json:"domains,omitempty"`
+	ExpiresAt   *string    `json:"expires_at,omitempty"`
```

Comment:
```suggestion
	ExpiresAt   *Timestamp    `json:"expires_at,omitempty"`
```

Reply:
This doesn't seem to work properly - The GitHub API is only returning a `YYYY-MM-DD` date and the `Timestamp` deserializer expects a full timestamp.  I looked at other places this library uses `Timestamp` and all of them appear to have the full date. Its a bit of a quirk with this API it seems. 

If we need it to be a timestamp we'll likely need to modify the Timestamp deserializer (which can be done, but seems a bit heavy-handed).  Happy to do whatever you think is best @gmlewis

Reply:
Thanks! Dropped a comment on the Struct